src/app/home-page/widget/entry-box/entry-box.component.html

```diff
@@ -20,7 +20,7 @@
       <div id="workflow-number" fxLayout="row">
         <mat-card-header fxLayout="row">
           <mat-card-title class="ml-2">{{ entryTypeLowerCase | titlecase }}s</mat-card-title>
-          <span class="bubble {{ entryTypeLowerCase }}-background">{{ totalEntries }}</span>
+          <span class="bubble {{ entryTypeLowerCase }}-background" data-cy="dashboard-entries-count-bubble">{{ totalEntries }}</span>
```

Comment:
Because this component is included multiple times, you're going to have multiple elements with this data-cy. Your tests work now because they're only loading notebook data, and we do have this pattern already, but you might want to consider prepending/inserting/appending entryTypeLowerCase to make the values more unique.

Reply:
Good catch! Fixed - thank you